tests/test_results_dialog_skin.py

```
@@ -73,6 +73,8 @@ def test_results_dialog_focused_row_has_high_contrast_focus_indicator():
 
 def _perceived_luminance(argb):
     """Rec.601 luma (0–255) of an 8-char ``AARRGGBB`` Kodi colour."""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)